addons/portal/controllers/portal_thread.py

```diff
@@ -72,8 +72,15 @@ def portal_message_fetch(self, thread_model, thread_id, fetch_params=None, **kw)
         domain = Domain.AND([
             self._setup_portal_message_fetch_extra_domain(kw),
             field.get_comodel_domain(model),
-            [('res_id', '=', thread_id), '|', ('body', '!=', ''), ('attachment_ids', '!=', False),
-             ("subtype_id", "=", request.env.ref("mail.mt_comment").id)]
+            [
+                ("res_id", "=", thread_id),
+                "|",
+                "&",
+                ("body", "!=", ""),
+                ("body", "!=", '<span class="o-mail-Message-edited"></span>'),
+                ("attachment_ids", "!=", False),
+                ("subtype_id", "=", request.env.ref("mail.mt_comment").id),
+            ],
```

Comment:
I find this syntax particularly confusing (already before this PR), I'd suggest to use `Domain` and & |  and parenthesis to better see what goes together with what.

Reply:
Doesn't it need an override in rating for the "empty" part to include messages that only have a rating?